modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsRelationshipsDisplayContext.java

```diff
@@ -210,6 +212,19 @@ public String getRESTContextPath(ObjectDefinition objectDefinition) {
 		return jaxRsApplicationDescriptor.getRESTContextPath();
 	}
 
+	public boolean isAddress() {
```

Comment:
@murilostodolni can we rename it to `isAddressSystemObject`? `isAddress` seems so generic..

Reply:
Yes, I will rename to `isAddressSystemObjectDefinition`